RHDEVDOCS-2072: Doc; [ODC] UI enhancements based on customer feedback (#61214)
openshift/openshift-docs: modules/odc-deploying-container-image.adoc

You can use an external image registry or an image stream tag from an internal registry to deploy an application on your cluster.

.Prerequisites

* You have logged in to the {product-title} web console and are in the *Developer* perspective.

.Procedure

. In the *+Add* view, click *Container images* to view the *Deploy Images* page.
. In the *Image* section:
.. Select *Image name from external registry* to deploy an image from a public or a private registry, or select *Image stream tag from internal registry* to deploy an image from an internal registry.
.. Select an icon for your image in the *Runtime icon* tab.
. In the *General* section:
.. In the *Application name* field, enter a unique name for the application grouping.
.. In the *Name* field, enter a unique name to identify the resources created for this component.
. In the *Resource type* section, select the resource type to generate:
.. Select *Deployment* to enable declarative updates for `Pod` and `ReplicaSet` objects.
.. Select *DeploymentConfig* to define the template for a `Pod` object, and manage deploying new images and configuration sources.
.. Select *Serverless Deployment* to enable scaling to zero when idle.

> . Click *Create*. You can see the build status of the application in the *Topology* view.

Reviewer: ```suggestion
. Click *Create*. You can view the build status of the application in the *Topology* view.
```
minor edit 🙂

Author: Made this change. Thanks!

// Module included in the following assemblies:
//
// * applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc

:_content-type: PROCEDURE
[id="odc-deploying-container-image_{context}"]
= Creating applications by deploying container image

> You can deploy existing images from an image registry or image stream tag to deploy it on to the {product-title}.

Reviewer: ```suggestion
You can use an external image registry or an image stream tag from an internal registry to deploy an image on your cluster.
```
suggestion for readability

Author: Made this change! Thanks.

.Prerequisites

* You have logged in to the {product-title} web console and are in the *Developer* perspective.

.Procedure

. In the *+Add* view, click the *Container images* link to view the *Deploy Images* page.
. In the *Image* section:

> .. Select *Image name from external registry*, to deploy an image from a private registry.

Author: Made this change! Thanks

> .. Select *Image name from external registry*, to deploy an image from a private registry.

Reviewer 2: ```suggestion
.. Select *Image name from external registry*, to deploy an image from a public or a private registry.
```

> .. Select *Image stream tag from internal registry*, to deploy an image from a public or a private registry.

Reviewer 2: ```suggestion
.. Select *Image stream tag from internal registry*, to deploy an image from an internal registry.
```

.. Select an icon for your image in the *Runtime icon* tab.
. In the *General* section:
.. In the *Application name* field, enter a unique name for the application grouping.
.. In the *Name* field, enter a unique name to identify the resources created for this component.

> . In the *Resources* section, select the resource type to generate:

Reviewer 2: ```suggestion
. In the *Resource type* section, select the resource type to generate:
```

.. Select *Deployment* to enable declarative updates for Pods and ReplicaSets.

> .. Select *Deployment Config* to define the template for a pod and manage deploying new images and configuration sources.

Author: Made this change! Thanks

> .. Select *Knative Service* to enable scaling to zero when idle.

Author: Made this change! Thanks

> . In the *+Add* view, click the *Container images* link to view the *Deploy Images* page.
> . In the *Image* section:
> .. Select *Image name from external registry*, to deploy an image from a public or a private registry.
> .. Select *Image stream tag from internal registry*, to deploy an image from an internal registry.
> .. Select an icon for your image in the *Runtime icon* tab.
> . In the *General* section:
> .. In the *Application name* field, enter a unique name for the application grouping.
> .. In the *Name* field, enter a unique name to identify the resources created for this component.
> . In the *Resource type* section, select the resource type to generate:
> .. Select *Deployment* to enable declarative updates for Pods and ReplicaSets.
> .. Select *DeploymentConfig* to define the template for a pod and manage deploying new images and configuration sources.
> .. Select *Serverless Deployment* to enable scaling to zero when idle.
> . Click *Create* to create the application. You can see the build status of the application in the *Topology* view.

Reviewer: ```suggestion
. In the *+Add* view, click *Container images* to view the *Deploy Images* page.
. In the *Image* section:
.. Select *Image name from external registry* to deploy an image from a public or a private registry, or select *Image stream tag from internal registry* to deploy an image from an internal registry.
.. Select an icon for your image in the *Runtime icon* tab.
. In the *General* section:
.. In the *Application name* field, enter a unique name for the application grouping.
.. In the *Name* field, enter a unique name to identify the resources created for this component.
. In the *Resource type* section, select the resource type to generate:
.. Select *Deployment* to enable declarative updates for `Pod` and `ReplicaSet` objects.
.. Select *DeploymentConfig* to define the template for a `Pod` object, and manage deploying new images and configuration sources.
.. Select *Serverless Deployment* to enable scaling to zero when idle.
. Click *Create*. You can see the build status of the application in the *Topology* view.
```

Author: Made this change. Thanks!
